Pipfile

```diff
@@ -52,11 +52,21 @@ typing-extensions = "~=4.4.0"
 greenlet = "~=2.0.1"
 
 # Old project requirements
-pymongo = "==3.7.1"
+pymongo = "==3.10.1"
 cherrypy = "==11.0.0"
 "dogpile.cache" = "==0.7.1"
 six = "==1.12.0"
 types-six = "==1.16.21.4"
+reproschema = "*"
+json5 = "==0.9.5"
+redis = "==3.5.2"
+ijson = ">=3.1.4"
+pycryptodomex = "*"
+cerberus = "==1.3.2"
+"backports-datetime-fromisoformat" = "==1.0.0"
+"backports.functools-lru-cache" = "==1.5"
+pandas = ">=1.5.3"
+tzlocal = ">=1.5.1"
```

Comment:
I added those deps one by one, fixing errors in the old project.
I'm not sure if it makes sense to refactor the old project, since we are going to remove old the project along with those deps after we import all data.

Reply:
Gotcha!